metabase/metabase
# Upgrade react-resizable and cleanup warnings

### Description

Cleanup warnings from browser console. They were caused by using older version of react-resizable. It's [been updated recently](https://github.com/react-grid-layout/react-resizable) with fixed prop types definition.

### How to verify

Since this change includes major version bump of the react-resizable, please make sure this functionality still works

### Demo

Just less warnings in the console

### Checklist



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29414)
